app/mm-redux/selectors/entities/preferences.ts

```diff
@@ -138,20 +138,65 @@ const getDefaultTheme = reselect.createSelector(getConfig, (config) => {
     return Preferences.THEMES.denim;
 });
 
-export const getTheme = createShallowSelector(
-    getThemePreference,
-    getDefaultTheme,
-    (themePreference, defaultTheme) => {
-        const themeValue: Theme | string = themePreference?.value ?? defaultTheme;
+const getLightThemePreference = createShallowSelector(
+    getCurrentTeamId,
+    getMyPreferences,
+    (currentTeamId, myPreferences) => {
+        return myPreferences[getPreferenceKey(Preferences.CATEGORY_THEME, currentTeamId)] ||
+            myPreferences[getPreferenceKey(Preferences.CATEGORY_THEME, '')];
+    },
+);
 
-        // A custom theme will be a JSON-serialized object stored in a preference
-        // At this point, the theme should be a plain object
-        const theme: Theme = typeof themeValue === 'string' ? JSON.parse(themeValue) : themeValue;
+export const getLightTheme = createShallowSelector(
+    getDefaultLightTheme,
+    getLightThemePreference,
+    (defaultLightTheme, lightThemePreference) => {
+        return normalizeTheme(lightThemePreference ? lightThemePreference.value : defaultLightTheme, defaultLightTheme);
+    },
+);
+
+const getDefaultDarkTheme = () => Preferences.THEMES.indigo;
```

Comment:
why this one does not uses a config value as the light one ? @matthewbirtch I thought the default dark theme was `onyx` I'm I mistaken ?